src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.Date.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Xunit;
+using Newtonsoft.Json;
+
+namespace System.Text.Json.Tests
+{
+    public static partial class Utf8JsonReaderTests
+    {
+        [Theory]
+        [MemberData(nameof(JsonDateTimeTestData.ValidISO8601Tests), MemberType = typeof(JsonDateTimeTestData))]
+        public static void TestingStringsConversionToDateTime(string jsonString, string expectedString)
```

Comment:
Is it safe to assume that this is just a test move (from the other file) with no modification?

**Edit:** Verified :)